src/Repository/Stubs/ProductVariantCartOrderItemStub.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusReserveStockPlugin\Repository\Stubs;
+
+use Setono\SyliusReserveStockPlugin\Repository\InCartQuantityForProductVariantOrderItemRepositoryAwareInterface;
+use Setono\SyliusReserveStockPlugin\Repository\ProductVariantCartOrderItem;
+use Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderItemRepository;
+
+final class ProductVariantCartOrderItemStub extends OrderItemRepository implements InCartQuantityForProductVariantOrderItemRepositoryAwareInterface
```

Comment:
Should this be included in the plugin or can it be done any other way? It seems it's only used in the specs and not in the actual plugin